use-cases/inferencing/benchmark/src/locust-tasks/logging.conf

```diff
@@ -0,0 +1,24 @@
+[formatters]
+keys=custom_json
+
+[formatter_custom_json]
+class=custom_json_formatter.CustomJSONFormatter
+
+[handlers]
+keys=console
+
+[handler_console]
+class=logging.StreamHandler
+args=(sys.stdout,)
+formatter=custom_json
+
+[loggers]
+keys=root
+
+[logger_root]
+level=INFO
+handlers=console
+
+[logger_model_eval]
```

Comment:
Your logger has to match your getLogger value

logger = logging.getLogger("benchmark_obj")